### PR TITLE
COMMERCE-10614 Refactor Price Modifier test case

### PR DESCRIPTION
**Pull request by baharturk**
https://issues.liferay.com/browse/COMMERCE-10614

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommercePriceLists.testcase [hunk 1]
```diff
@@ -1591,6 +1591,436 @@ definition {
...
+		}
+
+		task ("Then the price modifier CASE E is applied to the products on the Catalog Page") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium");
+
+			AnnouncementsPortlet.configurePaginationCP(itemsPerPage = 60);
+
+			var aux = 37;
+
+			for (var curr_productAndPrice : list ${productAndPriceArray}) {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommercePriceLists.testcase:1812]: Where is `productAndPriceArray` defined?

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:forward

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**